store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java

```diff
@@ -25,6 +25,10 @@
 
     public static final String MULTI_PATH_SPLITTER = System.getProperty("rocketmq.broker.multiPathSplitter", ",");
 
+    static {
+        System.setProperty("dLedger.multiPath.Splitter", MULTI_PATH_SPLITTER);
+    }
```

Comment:
Should we set this property when enbaleDLedgerCommitlog is true？

Reply:
@RongtongJin  Good suggestion!